scenes/game/local_game.gd

```diff
@@ -3477,7 +3501,8 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			_append_log_full(Enums.LogType.LogType_Effect, opposing_player, "is no longer prevented from moving.")
 			opposing_player.cannot_move = false
 		"bonus_action":
-			active_boost.action_after_boost = true
+			if not performing_player.no_extra_actions_or_strike_this_turn:
+				active_boost.action_after_boost = true
```

Comment:
this makes sense, but maybe also checking for active_strike would be helpful for futureproofing